docs/index.rst

```diff
@@ -66,6 +66,7 @@ Contributions are very welcome. If you have bug reports or feature requests or q
     :caption: User Documentation
 
     install
+    workflow
```

Comment:
workflows is not feature-complete yet so move it towards the end – otherwise people will read it first and expect that it already solves their problems